src/tests/api/CallbackApi.test.ts

```diff
@@ -7,7 +7,7 @@ import dataEeaIdCard from "../data/docSelectionPayloadEeaIdCardValid.json";
 import f2fStubPayload from "../data/exampleStubPayload.json";
 import integrationHappyPayload from "../data/integrationHappyPathPayload.json";
 import { postDocumentSelection, startStubServiceAndReturnSessionId, getSessionById, callbackPost, authorizationGet, tokenPost, userInfoPost, receiveJwtTokenFromSqsMessage, validateJwtToken } from "../utils/ApiTestSteps";
-
+import 'dotenv/config';
```

Comment:
yes - if you have a local file `src/.env` with KEY=VALUE pairs it will load these in to `process.env['KEY']=VALUE` so that the code runs with env vars set. If the file does not exist the line is silently ignored.